apps/edxapp/templates/mysql/job_00_load_sql_dump.yml.j2

```diff
@@ -1,42 +0,0 @@
-apiVersion: batch/v1
```

Comment:
You dump this we still need it for ginkgo no?.... :scream_cat:

Reply:
Yup. We'll never push ginkgo to production, so there will be no need to keep track of this legacy trick. That said, I've already tagged the "latest release" that used it: https://github.com/openfun/arnold/releases/tag/ginkgo.1-1.0.6